actor/v2action/service_access_test.go

```diff
@@ -625,6 +666,42 @@ var _ = Describe("Service Access", func() {
 			Expect(public).To(BeTrue())
 		})
 
+		When("the service is already enabled for orgs individually", func() {
+			BeforeEach(func() {
+				fakeCloudControllerClient.GetServicePlanVisibilitiesReturns(
+					[]ccv2.ServicePlanVisibility{
+						{GUID: "service-visibility-guid-1"},
+					},
+					nil, nil)
+			})
+
+			It("first disables the plan in both orgs", func() {
```

Comment:
This `It` description confused me - there is no reference to orgs anywhere in the test setup, and the method under test does not retrieve any orgs. It seems like the actual behavior is that for all plans for the target service, we will remove any associated service plan visibilities (the org associated with those visibilities doesn't matter). If this was intended to cover the case where multiple orgs have access to the plan, I would expect to see multiple service plan visibilities being returned.